script.sh

```diff
@@ -1,24 +1,26 @@
-#!/bin/sh
-
-version() {
-  if [ -n "$1" ]; then
-    echo "-v $1"
-  fi
-}
+#!/bin/sh -e
 
 cd "$GITHUB_WORKSPACE"
```

Comment:
```suggestion
cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
```

Reply:
Fixed in 27de346 by adding `workdir` option.